### PR TITLE
Use-Manage Buttons

### PR DESCRIPTION
**Pull request by ColinMcNeil**
# Release Notes

Restyle buttons with the UCONN design. Buttons colored based on use/manage mode.

# Screenshot
## Add Facility in use mode
screen shot 2018-12-04 at 8 29 32 am

## Add instrument in manage mode
screen shot 2018-12-04 at 8 29 48 am

## Hover state
Simply adding a 15% alpha layer
screen shot 2018-12-04 at 8 29 54 am

### CODE REVIEW

**Comment by jhanggi** [comment]
Is there anything in your styles for `btn-primary`? I feel like there should be something that calls makes those stand out a little more, especially when there is something like a "Save" and a "Cancel".

**Comment by ColinMcNeil** [comment]
Nothing in the design spec, but I agree so I gave them `font-weight: bold;` and `border-width: 1.5px;`. Stands out much better. Also gave them a different hover state.

File: app/assets/stylesheets/nucore.scss [hunk 1]
```diff
@@ -632,21 +632,33 @@ li:hover {
...
   text-shadow: none;
 }
 
+.btn-primary {
+  font-weight: bold;
+  border-width: 1.5px;
+}
+
 .btn:hover, .btn-add:hover {
+  transition: ease 0.2s all;
```
Comment (jhanggi) [inline app/assets/stylesheets/nucore.scss]: Do we need this? The `.btn` style feels like it should cover it.

**Comment by jhanggi** [comment]
This one also needs some merge conflict resolution.

File: app/assets/stylesheets/nucore.scss (hunk 1 above)
Comment (ColinMcNeil) [reply app/assets/stylesheets/nucore.scss]: Typically I wouldn't, but bootstrap is applying a hover transition so I had to override that.

Comment (jhanggi) [reply app/assets/stylesheets/nucore.scss]: Gotcha. If you can clean up the conflicts, I'll merge it!

**Comment by ColinMcNeil** [comment]
Should be good

**Pull request merged**